go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -96,6 +97,8 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) func
 		exit, err := f(c, log, cmdName)
 		if err != nil {
 			log.Error("ExitErrAction encountered error. err:%s", err)
+			// Print error message to the user.
+			fmt.Fprintf(os.Stderr, "fatal: %s\n", err)
```

Comment:
Would you also update places where we print explicitly (like credential.go) to not have duplicate entries?